AUTHORS

```diff
@@ -623,6 +623,9 @@ ScummVM Team
     Tizen / BADA:
        Chris Warren-Smith
 
+    Webassembly / Emscripten:
```

Comment:
Updating the credits is normally done by editing `devtools/credits.pl`, followed by running `make credits` to update the other files.

Reply:
I updated `devtools/credits.pl` as well and ran `make credits`. Thank you for the hint!